### PR TITLE
New package: chaparral - radiation view factor library

### PR DESCRIPTION
**Pull request by pbrady**

### CODE REVIEW

File: var/spack/repos/builtin/packages/chaparral/package.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+class Chaparral(CMakePackage):
+    """Radiation view factor library"""
+
+    homepage = "https://gitlab.com/truchas/tpl-forks/chaparral"
+    git      = "https://gitlab.com/truchas/tpl-forks/chaparral.git"
+
+    maintainers = ['pbrady']
+
+    version('develop', branch='truchas')
+    version('develop-2020-08-28',
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/chaparral/package.py]:
```suggestion
    version('2020-08-28',
```

File: var/spack/repos/builtin/packages/chaparral/package.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Chaparral(CMakePackage):
+    """Radiation view factor library"""
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/chaparral/package.py:10]: The repo description mention this is a fork of an equally named project from Sandia, can you clarify that a bit? I am trying to understand if we need to reserve the "pristine" name for the original software. Thanks!

**Review by alalazo — CHANGES_REQUESTED** [review]
I have a question and a suggestion, but other than that LGTM.

File: var/spack/repos/builtin/packages/chaparral/package.py (hunk 1 above)
Comment (pbrady) [reply var/spack/repos/builtin/packages/chaparral/package.py]: Thanks for the review.  I've remove the 'develop' prefix as suggested.

File: var/spack/repos/builtin/packages/chaparral/package.py (hunk 2 above)
Comment (pbrady) [reply var/spack/repos/builtin/packages/chaparral/package.py:10]: The project originated with SNL in 1995 and they sent us a lgpl'd version about 20 years ago.  From what I've heard, they had plans to release it but never did.  

My goal here is to prepare a spack recipe for an open source casting code https://gitlab.com/truchas/truchas

With that in mind, if in the future SNL does release the "true" version of chaparral, I would be fine with adjusting this package.py to default to their repo and use our forked repo as a "truchas" version.

Does that work?

Comment (alalazo) [reply var/spack/repos/builtin/packages/chaparral/package.py:10]: That works perfectly. Thanks!

**Review by alalazo — DISMISSED** [review]

**Comment by alalazo** [comment]
Can you rebase to run Gitlab pipelines on a new merge commit?

**Comment by pbrady** [comment]
rebased

**Review by alalazo — APPROVED** [review]

**Pull request merged**